insight.py

```diff
@@ -31,18 +31,19 @@
 from measurement.files import logfiles
 from measurement.process import meta as proc_meta
 from measurement.tidb import pdctl
-
+from measurement.ftrace import ftrace
 
 class Insight():
     # data output dir
-    outdir = "data"
+    outdir = "%s/data" % os.getcwd()
```

Comment:
It's not necessary to `getcwd()` here, as in `fileutils.create_dir()` we use `os.makedirs()` to create/check target directory, which uses the (main) process's cwd for relative paths by default.

Reply:
If we call `fileutils.create_dir()` after `chdir()` with `outputdir=None`, the relative paths will not be the (main) process's cwd.